hw/top_earlgrey/ip_autogen/alert_handler/data/alert_handler.hjson

```diff
@@ -327,7 +327,47 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "CSR.SHADOW",
+      desc: "Important CSRs are shadowed."
+    }
+    { name: "ALERT.SIGNAL.DIFF",
+      desc: "Differentially encoded alert channels."
+    }
+    { name: "LPG.SIGNAL.MUBI",
+      desc: "LPG signals are encoded with MUBI types."
+    }
+    { name: "ESC.SIGNAL.DIFF",
+      desc: "Differentially encoded escalation channels."
+    }
+    { name: "ALERT_RX.SIGNAL.BKGN_CHK",
+      desc: "Periodic background checks on alert channels (ping mechanism)."
+    }
+    { name: "ESC_TX.SIGNAL.BKGN_CHK",
+      desc: "Periodic background checks on escalation channels (ping mechanism)."
+    }
+    { name: "ESC_RX.SIGNAL.BKGN_CHK",
+      desc: "Escalation receivers can detect absence of periodic ping requests."
+    }
+    { name: "ESC_TIMER.FSM.SPARSE",
+      desc: "Escalation timer FSMs are sparsely encoded."
+    }
+    { name: "PING_TIMER.FSM.SPARSE",
+      desc: "Ping timer FSM is sparsely encoded."
+    }
```

Comment:
Is there also some `LOCAL_ESC` that should be listed here?

Reply:
Hm good point - they should have a local escalation action indeed. Let me add that.